Solution1/Solution1.Tests/Infrastructure/Extensions/HttpMessageHandlerMockExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Moq;
+using Moq.Protected;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Weather.Tests.Infrastructure.Extensions
+{
+    public static  class HttpMessageHandlerMockExtensions
+    {
+        public static void GetSettings(this Mock<HttpMessageHandler> httpMessageHandler, HttpResponseMessage response, string uri)
```

Comment:
you return void - why method called GetSetting? bad idea to put it in separate class, return to tests

Reply:
+